taxcalc/simpletaxio.py

```diff
@@ -0,0 +1,648 @@
+"""
+OSPC taxcalc package simple tax input-output class.
+"""
+# CODING-STYLE CHECKS:
+# pep8 --ignore=E402 simpletaxio.py
+# pylint --disable=locally-disabled simpletaxio.py
+
+import os
+import sys
+import json
+import re
+import numpy.core as np
```

Comment:
the standard idiom here would be
```
import numpy as np
```
There is only one usage of the `np` alias below, referring to `numpy.core.array`, which is the same as `numpy.array`, but additional uses of `np` by other developers would likely produce unexpected results.